mashups/other/gen8flaaapped.tour

```diff
@@ -2,4 +2,4 @@
 /tour name Flipped Almost Any Ability
 /tour autodq 3
 /tour autostart 10
-/tour rules flippedmod, +Kartana, +Dragapult, +Pheromosa, +Melmetal, +Zeraora, +Zygarde-Base, +Keldeo, +Buzzwole, +Urshifu-Rapid-Strike, +Urshifu-Single-Strike, +Spectrier, -Regice, -Regirock, +Dracovish, +Naganadel, +Marshadow, +Dragonite
+/tour rules flippedmod, +Kartana, +Dragapult, +Pheromosa, +Melmetal, +Zeraora, +Zygarde-Base, +Keldeo, +Buzzwole, +Urshifu-Rapid-Strike, +Urshifu-Single-Strike, +Spectrier, -Regice, -Regirock, +Dracovish, +Naganadel, +Marshadow, +Dragonite, +Yveltal
```

Comment:
Actually ignore this unban, will be making more significant changes to banlist sometime in the near future.